atlp-rwanda/e-comm-team-techsmith-fn
# #185218917 Users should be presented with a reset password link

## What does this PR do?
Users should be presented with a reset password link


## Description of Task to be completed?
- In case the users password is forgotten, functionality should exists to reset the password via an associated and email account.
- Users should be presented with a reset password link on the login page, there should also be another page for validating token sent via email and new password and confirm password should be inputted and validated.
## How should this be manually tested?

- Clone the repository using `git clone`  [link](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn)
-  Install node packages that the application depends on: `npm install`
-  Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.
- Start the application development server: `npm run dev`
-  Check out the components documentation: `npm run storybook`

- Checkout to the branch using `git checkout ft-Users-should-be-presented-with-a-reset-password-link-185218917 `

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run dev`

- Check the application's documentation on Storybook: `npm run storybook`

## Any background context you want to provide?

## What are the relevant pivotal tracker/Trello stories?

## Screenshots (if appropriate)

## Questions:
